--- a/vendor/api-platform/core/src/Doctrine/Orm/Filter/SearchFilter.php	2022-10-07 13:34:55.000000000 +0200
+++ b/vendor/api-platform/core/src/Doctrine/Orm/Filter/SearchFilter.php	2022-11-06 16:15:27.960051980 +0100
@@ -35,7 +35,7 @@
  *
  * @author Kévin Dunglas <dunglas@gmail.com>
  */
-final class SearchFilter extends AbstractFilter implements SearchFilterInterface
+class SearchFilter extends AbstractFilter implements SearchFilterInterface
 {
     use SearchFilterTrait;
 
