diff --git a/vendor/api-platform/core/src/Serializer/AbstractItemNormalizer.php b/patches/tmp/AbstractItemNormalizer.php
index 71eb31190..a62a4befc 100644
--- a/vendor/api-platform/core/src/Serializer/AbstractItemNormalizer.php
+++ b/patches/tmp/AbstractItemNormalizer.php
@@ -99,7 +99,7 @@ abstract class AbstractItemNormalizer extends AbstractObjectNormalizer
      *
      * @return bool
      */
-    public function supportsNormalization($data, $format = null)
+    public function supportsNormalization($data, $format = null, array $context = [])
     {
         if (!\is_object($data)) {
             return false;
@@ -180,7 +180,7 @@ abstract class AbstractItemNormalizer extends AbstractObjectNormalizer
      *
      * @return bool
      */
-    public function supportsDenormalization($data, $type, $format = null)
+    public function supportsDenormalization($data, $type, $format = null, array $context = [])
     {
         return $this->localCache[$type] ?? $this->localCache[$type] = $this->resourceClassResolver->isResourceClass($type);
     }
