--- a/vendor/symfony/validator/Context/ExecutionContext.php	2022-04-15 08:08:08.000000000 +0000
+++ b/vendor/symfony/validator/Context/ExecutionContext.php	2022-05-27 13:12:18.592663850 +0000
@@ -44,8 +44,8 @@
      */
     private mixed $root;
 
-    private TranslatorInterface $translator;
-    private ?string $translationDomain;
+    public TranslatorInterface $translator;
+    public ?string $translationDomain;
 
     /**
      * The violations generated in the current context.
