--- a/vendor/api-platform/core/src/Metadata/Property/Factory/AnnotationPropertyNameCollectionFactory.php	2022-01-11 10:29:54.000000000 +0000
+++ b/vendor/api-platform/core/src/Metadata/Property/Factory/AnnotationPropertyNameCollectionFactory.php	2022-03-24 09:17:39.399960873 +0000
@@ -67,7 +67,7 @@
         // Properties
         foreach ($reflectionClass->getProperties() as $reflectionProperty) {
             if (
-                (\PHP_VERSION_ID >= 80000 && $reflectionProperty->getAttributes(ApiProperty::class)) ||
+                (\PHP_VERSION_ID >= 80000 && $reflectionProperty->getAttributes(ApiProperty::class, \ReflectionAttribute::IS_INSTANCEOF)) ||
                 (null !== $this->reader && null !== $this->reader->getPropertyAnnotation($reflectionProperty, ApiProperty::class))
             ) {
                 $propertyNames[$reflectionProperty->name] = $reflectionProperty->name;
